src/app/beer_garden/api/http/client.py

```diff
@@ -1,23 +1,38 @@
 # -*- coding: utf-8 -*-
 import json
+import logging
 from inspect import isawaitable
 
 import six
-from brewtils.models import BaseModel
+from brewtils.models import BaseModel, Operation
 from brewtils.schema_parser import SchemaParser
 
 import beer_garden.api
 import beer_garden.router
 
+logger = logging.getLogger(__name__)
+
 
 class SerializeHelper(object):
-    async def __call__(self, *args, serialize_kwargs=None, **kwargs):
-        result = beer_garden.router.route(*args, **kwargs)
+    async def __call__(self, operation: Operation, serialize_kwargs=None, **kwargs):
+        result = beer_garden.router.route(operation, **kwargs)
 
         # Await any coroutines
         if isawaitable(result):
             result = await result
 
+        # This is a helper function to find bad formatted Operations/Clients
+        if (
+            serialize_kwargs is None
+            and operation.kwargs is not None
+            and "serialize_kwargs" in operation.kwargs
+        ):
+            serialize_kwargs = operation.kwargs["serialize_kwargs"]
+            logger.info(
+                f"Serialized Kwargs was pass through Operation instead of Client "
+                f"function for {operation}, this was migrated but should be fixed"
```

Comment:
I don't think this is a good idea. You're adding a non-trivial check to everything coming through the REST API at runtime to catch a programming error. If we want to *make sure* that this never happens then the right place to do that is in our tests.

I'm not convinced that's necessary, though. Unless we start seeing this happen all over the place it's probably not worth the effort. But I definitely don't think it's worth the cost of this check.

Reply:
I agree, so I will remove it because we fixed it in the API